oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -1,37 +1,37 @@
 /**
  * Oshi (https://github.com/oshi/oshi)
- *
+ * <p>

```

Comment:
These comment headers are automatically rewritten by maven.  If you need to change them, you should change the [LICENSE_HEADER](https://github.com/oshi/oshi/blob/master/LICENSE_HEADER) file.